repository/channel_impl_test.go

```diff
@@ -148,7 +151,36 @@ func TestGormRepository_GetChannelStats(t *testing.T) {
 	t.Run("success", func(t *testing.T) {
 		t.Parallel()
 
+		channel := mustMakeChannel(t, repo, rand)
+		user1 := mustMakeUser(t, repo, rand)
+		user2 := mustMakeUser(t, repo, rand)
+		u1Stamp1 := mustMakeStamp(t, repo, rand, user1.GetID())
+		u2Stamp1 := mustMakeStamp(t, repo, u1Stamp1.Name, user2.GetID())
```

Comment:
スタンプは同じ名前で二つ作ることはできないです。
参照: https://github.com/traPtitech/traQ/blob/master/docs/dbschema/stamps.md のunique key
```suggestion
		u1Stamp1 := mustMakeStamp(t, repo, rand, user1.GetID())
		u2Stamp1 := mustMakeStamp(t, repo, rand, user2.GetID())
```